### ISSUE TITLE
Nvda freezes in the save as dialog in microsoft office 2010

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2960]
**Reported by zvonac123 on 2013-02-03 18:35**
I'am using 64 bit windows 7 and the latest snapshot. Nvda freezes in the microsoft office 2010 save as dialog. I have installed the latest updates in windows 7.

**Comment by leonardder** [comment]
Anyone able to reproduce this with office 2010? This ticket looks quite abandoned. If we haven't received more information about this at the end of next week, I will close it due to lack of enough information. For example, it would have helped if we knew whether the issue also occurred with NVDA 2012.4
